port_ocean/tests/core/handlers/entity_processor/test_jq_entity_processor.py

```diff
@@ -269,3 +269,35 @@ async def test_parse_items_performance_10000(
             assert len(result.entity_selector_diff.passed) == 1
             assert result.entity_selector_diff.passed[0].properties.get("foo") == "bar"
             assert not result.errors
+
+    async def test_parse_items_wrong_mapping(
+        self, mocked_processor: JQEntityProcessor
+    ) -> None:
+        mapping = Mock()
+        mapping.port.entity.mappings.dict.return_value = {
+            "identifier": ".ark",
+            "blueprint": ".baz",
+            "properties": {"foo": ".bazbar"},
+            "title": ".foo",
+            "desc": ".foobar",
+            "name": ".bar.baz",
```

Comment:
lets follow mapping conventions 

there is no such keys as `name` and `desc` I would rather that you add it to properties and relations

Reply:
maybe worth adding how it would be with `items_to_parse` ?

Reply:
used the properties under the mapping property from [our docs](https://docs.getport.io/build-your-software-catalog/customize-integrations/configure-mapping/)